### PR TITLE
[61364115] Updated questionnaire to be the usc-chan questionnaire

### PR DESCRIPTION
**Pull request by emerencia**
Update the questionnaire according to https://docs.google.com/document/d/1kz77OAQBI38Ite9_qILyoiq0oKSNHQbNAoMaiwlMsuY/edit

Fixes https://freedcamp.com/view/3448368/milestones/panel/task/61364115

screencapture-localhost-3002-questionnaire-c4f5b8b3-2693-427f-a4ca-9bae488a31c1-2024-10-14-22_20_22

### CODE REVIEW

File: projects/usc-chan/seeds/questionnaires/usc_chan.rb [hunk 1]
```diff
@@ -8,100 +8,399 @@
...
     id: :v2,
     hidden: true,
-    type: :range,
-    title: 'Example with a range',
-    labels: ['very little', 'very much']
+    required: true,
+    type: :time,
+    title: 'What time did you go to bed?',
+    hours_from: 0,
+    hours_to: 24, # TODO: fix AM/PM notation for the US.
```
Comment (GarryBell) [inline projects/usc-chan/seeds/questionnaires/usc_chan.rb:31]: I think this is fixed in this PR?

File: projects/usc-chan/seeds/questionnaires/usc_chan.rb [hunk 2]
```diff
@@ -8,100 +8,399 @@
...
+    id: :v33,
+    required: true,
+    type: :range,
+    title: 'Right now, how <strong>bothered</strong> are you by your diabetes or its management?',
+    labels: ['Not at all bothered', 'Extremely bothered'],
+  }, {
+    id: :v34,
+    required: true,
+    type: :range,
+    title: 'Right now, how <strong>tired</strong> do you feel?',
```
[the remark this answers is not in the document]
Comment (GarryBell) [reply projects/usc-chan/seeds/questionnaires/usc_chan.rb:375]: Also they don't show up in Firefox for me

File: projects/usc-chan/seeds/questionnaires/usc_chan.rb (hunk 1 above)
Comment (emerencia) [reply projects/usc-chan/seeds/questionnaires/usc_chan.rb:31]: Fixed.

File: projects/usc-chan/seeds/questionnaires/usc_chan.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (emerencia) [reply projects/usc-chan/seeds/questionnaires/usc_chan.rb:375]: This is really weird, they do show up for me, also in Chrome.

[the remark this answers is not in the document]
Comment (emerencia) [reply projects/usc-chan/seeds/questionnaires/usc_chan.rb:375]: Ok, I fixed it. The font weight 500 apparently doesn't show up everywhere as bold. Nice catch.

File: projects/usc-chan/seeds/questionnaires/usc_chan.rb [hunk 3]
```diff
@@ -8,100 +8,399 @@
...
   }, {
     id: :v4,
-    type: :date,
-    title: 'Example of a date question',
-    labels: ['completely intuitive', 'completely planned']
+    hidden: true,
+    required: true,
+    type: :range,
+    title: 'Right now, how well rested do you feel?',
+    labels: ['Not al all rested', 'Extremely rested'], # TODO: say we can do sliders without initial thumb
```
[the remark this answers is not in the document]
Comment (emerencia) [reply projects/usc-chan/seeds/questionnaires/usc_chan.rb:54]: Fixed.

**Pull request merged**